src/view/global.styl

```diff
@@ -23,6 +23,16 @@ code {
   font-family: monospace;
 }
 
+.u-toolTipMinimumSize {
+  min-width:  120*$px !important;
+}
+
+// This provides enough padding to the right of the 'learn more' link on events
+// to allow the error toolTip to dispaly on the right side of the text field
+.u-gapLinkRight {
+  margin-right: $gap*7 !important;
```

Comment:
Let's use a hard-coded width with like you did with `toolTipMinimumSize`.

Reply:
Style was removed and made part of a component. :)